packages/owl-bot/src/handlers.ts

```diff
@@ -335,3 +342,95 @@ Please fix this as soon as possible so that your repository will not go stale.`;
     );
   }
 }
+
+/**
+ * Invoked when someone edits the title of body of a pull request.
+ * We're interested if they checked the box to "regenerate this pull request."
+ */
+export async function handlePullRequestEdited(
+  appId: number,
+  privateKey: string,
+  project: string,
+  trigger: string,
+  payload: PullRequestEditedEvent,
+  octokit: OctokitType,
+  logger = console
+) {
+  const head = payload.pull_request.head.repo.full_name;
```

Comment:
I would be tempted to do this data munging in `owl-bot.ts`, then `handlePullRequestEdited` could just accept the parameters that it needs, and it doesn't need to know about the full `payload` structure.

Reply:
Done.